tensorflow_app/views/export_graphdef.py

```diff
@@ -1,58 +1,196 @@
+                stack.append(layerId)
+            if (not net[layerId]['connection']['input']):
+                inputLayerId.append(layerId)
+            if (not net[layerId]['connection']['output']):
+                outputLayerId.append(layerId)
+        if len(error):
+            return JsonResponse(
+                {'result': 'error', 'error': 'Cannot convert ' + ', '.join(error) + ' to Keras'})
+
+        while(len(stack)):
+            if ('Loss' in net[layerId]['info']['type'] or
+                    net[layerId]['info']['type'] == 'Accuracy'):
+                pass
+            elif (net[layerId]['info']['type'] in layer_map):
+                i = len(stack) - 1
+                while isProcessPossible(stack[i]) is False:
+                    i = i - 1
+
+                layerId = stack[i]
+                stack.remove(layerId)
+                if (net[layerId]['info']['type'] != 'Scale'):
+                    layer_in = [net_out[inputId]
+                                for inputId in net[layerId]['connection']['input']]
+                # Need to check if next layer is Scale
+                if (net[layerId]['info']['type'] == 'BatchNorm'):
+                    idNext = net[layerId]['connection']['output'][0]
+                    nextLayer = net[idNext]
+                    # If the BN layer is followed by Scale, then we need to pass both layers
+                    # as in Keras parameters from both go into one single layer
+                    net_out.update(layer_map[net[layerId]['info']['type']](
+                        net[layerId], layer_in, layerId, idNext, nextLayer))
+                elif (net[layerId]['info']['type'] == 'Scale'):
+                    type = net[net[layerId]['connection']
+                               ['input'][0]]['info']['type']
+                    if (type != 'BatchNorm'):
+                        error.append(
+                            layerId + '(' + net[layerId]['info']['type'] + ')')
+                else:
+                    net_out.update(layer_map[net[layerId]['info']['type']](
+                        net[layerId], layer_in, layerId))
+                for outputId in net[layerId]['connection']['output']:
+                    if outputId not in stack:
+                        stack.append(outputId)
+                processedLayer[layerId] = True
+            else:
+                error.append(
+                    layerId + '(' + net[layerId]['info']['type'] + ')')
+        if len(error):
+            return JsonResponse(
+                {'result': 'error', 'error': 'Cannot convert ' + ', '.join(error) + ' to Keras'})
+
+        final_input = []
+        final_output = []
+        for i in inputLayerId:
+            final_input.append(net_out[i])
+
+        for j in outputLayerId:
+            final_output.append(net_out[j])
+
+        model = Model(inputs=final_input, outputs=final_output, name=net_name)
+        json_string = Model.to_json(model)
```

Comment:
It would be better if we can reuse the code from `export_json` instead of maintaining it in two places.

Reply:
That would need a minor change at both the places